packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -1269,6 +1269,47 @@ describe("TableComponent", () => {
 		});
 	});
 
+	describe("setCellFormatter", () => {
+		const dummyData: object[] = [
+			{ id: 1, description: "dummy 1", test: "test-1" },
+			{ id: 2, test: "test-2" },
+			{ id: 3, description: "dummy 3" }
+		];
+
+		beforeEach(() => {
+			hostComponent.columnProperties = [
+				{ name: "id", cellFormatter: (value: any): string => (value === 1 ? "one" : "") },
+				{ name: "description", cellFormatter: (value: any): string => typeof value === "undefined" ? "-null-" : value },
+				{ name: "test" }
+			];
+			hostComponent.dummyData = dummyData;
+
+			hostFixture.detectChanges(); // trigger data binding
+			component.ngAfterViewInit();
+		});
+		
+		it("should display 'ONE' instead of '1' when id == '1'", () => {
```

Comment:
Let's put in the test description only the functionality tested instead of the specific details... for example: `should displayed the formatted value in the cell instead of the raw value`